common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ShuffleIndexInformation.java

```diff
@@ -53,9 +53,9 @@ public int getSize() {
   /**
    * Get index offset for a particular reducer.
    */
-  public ShuffleIndexRecord getIndex(int reduceId) {
-    long offset = offsets.get(reduceId);
-    long nextOffset = offsets.get(reduceId + 1);
+  public ShuffleIndexRecord getIndex(int startReduceId, int endReduceId) {
```

Comment:
This is a public method, do we need to add a new method instead to achieve binary compatibility?

Reply:
Thanks, done in 8a323ab.